Price Calculator/Product.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Collections.Generic;
+using static Price_Calculator.DiscountInformation;
+
+
+namespace Price_Calculator
+{
+    public class Product
+    {
+        public string Name { get; set; }
+        public int UPC { get; set; }
+        public decimal Price { get; set; }
+        public string Currency { get; set; } = "USD";
+        public static decimal Tax { get; set; } = 20;
```

Comment:
no need to make it static

Reply:
Since the Tax applied to all Products, not to specific product so I made it static 
(The client uses this property by Class name not Instance name)